### DISCUSSION TITLE
Transaction on ethurium mainet

### DISCUSSION THREAD

**Discussion by sagoto80** [Cyfrin/foundry-full-course-cu#1693] [Q&A] [answered]
https://ethereum.org/en/developers/docs/transactions/
>

The vast majority of transactions access a contract from an externally-owned account.isnt all transaction initited by eoa

**Comment by EngrPips** [comment, marked as answer]
Abstracted Account (AA) can also initiate a transaction based on an initial call from an eoa. You can read up about them. This link, https://blog.thirdweb.com/account-abstraction-erc4337/, can help you to get started.
